src/posts/2023-04-03-eu-talks/index.md

```diff
@@ -0,0 +1,130 @@
+---
+path: '/blog/2023/04/03/cilium-kubecon-eu-2023-talks'
+date: '2023-04-03T17:00:00.000Z'
+title: 'Cilium Talks at KubeCon EU 2023'
+isPopular: true
+isFeatured: false
+ogImage: talkseu.png
+ogSummary: "Find all the Cilium talk at KubeCon EU 2023"
+categories:
+  - Community
+tags:
+  - Cilium
+  - KubeCon
+---
+
+![Cilium Talks at KubeCon](talkseu.png) 
+
+*Author: Bill Mulligan, Isovalent*
+
+KubeCon + CloudNativeCon EU 2023 is right around the corner and it is promising to be a hive of activity around Cilium (all of those tulips won’t pollinate themselves). The first ever [CiliumCon](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/co-located-events/ciliumcon/) is happening with talks from users like Bloomberg, New York Times, and Robinhood. Tickets are already completely sold out, but there will be a livestream. You will also be able to join the conversation in the Cilium slack by jumping in the [#ciliumcon channel](https://cilium.slack.com/archives/C0510801E4V).
+
+Head to the [project meeting](https://docs.google.com/document/d/18DjEz8MzvhWkz7Y7ghWYW3Km_wCmmaZ4Z4iurwFgnYk/edit#) to meet the maintainers and discuss the future of the project. If day one hasn’t already tired you out, there will be 7 other talks about Cilium to catch when the main conference kicks off on Wednesday!
```

Comment:
```suggestion
Head to the [project meeting](https://docs.google.com/document/d/18DjEz8MzvhWkz7Y7ghWYW3Km_wCmmaZ4Z4iurwFgnYk/edit#) on Tuesday afternoon to meet the maintainers and discuss the future of the project. 

If that action-packed first day hasn’t already tired you out, there will be 7 other talks about Cilium to catch when the main conference kicks off on Wednesday!
```

Reply:
Just suggesting adding in some more sign-posting of the timing (hoping that people hold “Tuesday” in their heads after reading this)